cmd/ndm_daemonset/probe/udevprobe.go

```diff
@@ -360,3 +364,79 @@ func (up *udevProbe) free() {
 		up.udevEnumerate.UnrefUdevEnumerate()
 	}
 }
+
+func (up *udevProbe) listenUdevEventMonitor() {
+	for {
+		event := <-up.udeveventSubscription.Events()
+		controller.EventMessageChannel <- processUdevEvent(event)
+	}
+}
+
+func processUdevEvent(event udevevent.UdevEvent) controller.EventMessage {
+	defer event.UdevDeviceUnref()
+	diskInfo := make([]*blockdevice.BlockDevice, 0)
+	uuid := event.GetUid()
+	path := event.GetPath()
+	action := event.GetAction()
+	klog.Infof("processing new event for (%s) action type %s", path, action)
+	deviceDetails := &blockdevice.BlockDevice{}
+	deviceDetails.UUID = uuid
```

Comment:
provide a comment here. This uuid will be the legacy format. We will be overwriting this UUID in the event handler.